packages/govern-core/contracts/pipelines/GovernQueue.sol

```diff
@@ -71,8 +71,8 @@ contract GovernQueue is IERC3000, AdaptativeERC165, IArbitrable, ACL {
 
     function initialize(address _aclRoot, ERC3000Data.Config memory _initialConfig) public initACL(_aclRoot) onlyInit("queue") {
         _setConfig(_initialConfig);
-        _registerStandard(ARBITRABLE_INTERFACE_ID);
-        _registerStandard(ERC3000_INTERFACE_ID);
+        _registerStandard(type(IArbitrable).interfaceId);
```

Comment:
How does this work? Does Solidity automatically generate this for us?

But also, there should literally be no interfaces resulting from this. `IArbitrable` doesn't specify any functions to implement.